app/assets/stylesheets/shared/_navigations.scss

```diff
@@ -1,71 +1,592 @@
+// Logo 
+
+.logo {
+    margin-left: 9px; 
+    margin-top: 9px;
+    position: absolute;
+    transition: width 150ms ease-out;
+    width: 175px;
+
+    @include breakpoint($screen-xxs){
+        margin-left: 15px; 
+        margin-top: 4px;
+        width: 210px;
+    }
+
+    @include breakpoint($screen-xs){
+        margin-top: 2px;
+        width: 235px;
+    }
+
+    @include breakpoint($screen-sm){
+        margin-top: 0;
+        padding-left: 0;
+        width: 300px;
+    }
+}
+
+
+// Masthead Header component
+
 .masthead {
     @extend .container-fluid;
     
     &__header {
         @include make-row();
+        display: none;
         padding: 20px 0;
+
+        @include breakpoint($screen-sm){
+            display: block;
+        }
+
+
+        &--sm {
+            @include breakpoint($screen-sm){
+                display: none;
+            }
+        }
+    }
+
+    &__brand {
+        @include make-xs-column(6);
+        @include make-sm-column(6);
+        @include make-md-column(7);  
+    }
+
+    &__actions {
+        @include make-xs-column(6);
+        @include make-sm-column(6);
+        @include make-md-column(5);
+        
+        margin-top: 7px;
+        text-align: right;
+        
+        @include breakpoint($screen-sm){
+            margin-top: 12px;
+        }
+
+        .button__login {
+            @extend .button__primary;
+            font-size: 85%;
+
+            @include breakpoint($screen-xs){
+                font-size: 100%;
+            }
+        }
+
+        .button__signup {
+            @extend .button__secondary;
+            display: none;
+
+            @include breakpoint($screen-xs){
+                display: inline-block;
+                margin-right: 5px;
+            }
+        }
+    }
+}
+
+
+// Setup for Flyout Navigation
+
+html,
+body,
+.wrap,
+.wrapper-push,
+.wrapper-content {
+    height: 100%;
+}
+
+.wrap {
+    overflow: hidden;
+    perspective: 1500px;
+    perspective-origin: 0% 50%;
+    position: relative;
+}
+
+.wrapper-push {
+    height: 100%;
+    left: 0;
+    position: relative;
+    transform-style: preserve-3d;
+    transition: transform 0.4s;
+    z-index: 99;
+    
+    .wrap.is-active & {
+        transform: translate3d(-300px, 0, 0);
+        
+        @include breakpoint($screen-sm) {
+            transform: none;
+        }
+    }
+    
+    &:after {
+        background: rgba(0,0,0,0.2);
+        content: '';
+        height: 0;
+        opacity: 0;
+        position: absolute;
+        right: 0;
+        top: 0;
+        transition: opacity 0.5s, width 0.1s 0.5s, height 0.1s 0.5s;
+        width: 0;
+        
```

Comment:
@heyitsolivia there is a bourbon mixin to handle triangles http://bourbon.io/docs/#triangle 
Not saying that we have to use.  Just pointing it out